src/components/image/Image.tsx

```diff
@@ -0,0 +1,27 @@
+import React from 'react'
+import { ImageProps } from './Image.types'
+import {
+  StyledImage,
+  ImageContainer,
+  StyledFigure,
+  ImageWrapper,
+  StyledFigureCaption,
+} from './Image.styles'
+
+export const Image: React.FC<ImageProps> = ({
+  image,
+  alt,
+  caption,
+  ...props
+}) => {
+  return (
+    <StyledFigure>
+      <ImageContainer>
+        <ImageWrapper>
+          <StyledImage fixed={image} alt={alt} {...props} />
```

Comment:
We'll want the pass-through of `props` to be on the root of the component, so that any extra stylings can applied at that level 😄